src/modules/heatmap/main.ts

```diff
@@ -17,7 +17,7 @@ export default <ModuleMainFunction>(async ({
     $m,
     setSetting,
 }) => {
-    if (window.location.pathname === '/' && window.hasOwnProperty('mapkit'))
+    if (window.location.pathname !== '/' || window.hasOwnProperty('mapkit'))
```

Comment:
this will cause heatmap not to be available on profiles anymore => please undo or remove the pathname condition (mapkit is only on the `/` path)